packages/angular_devkit/build_angular/src/builders/ssr-dev-server/index.ts

```diff
@@ -220,7 +220,7 @@ function startNodeServer(
 ): Observable<void> {
   const outputPath = serverOutput.outputPath as string;
   const path = join(outputPath, 'main.js');
-  const env = { ...process.env, PORT: '' + port };
+  const env = { ...process.env, SSR_PORT: '' + port };
```

Comment:
In the dev-server we need to support both for backwards computability. 

```suggestion
  const env = { ...process.env, SSR_PORT: '' + port, PORT: '' + port };
```